readme.md

```diff
@@ -38,35 +38,34 @@ Once installation is working, take a look at the existing code to make sure you
 
 * **Question:** Describe what this code is doing and what its purpose is.
 
-* **Your Answer:** 
+* **Your Answer:** This code is bringing in the Party model and clearing all Party records with an asynchronous reset() function. Afterwards, it creates two parties. Overall, it resets the Party records in mongo-db.
 
 ---
 
 - [ ] Imagine that as a user, you enter your username and password into a site in order to signup.
 
 * **Question:** What happens next?
 
-* **Your Answer:**
-
+* **Your Answer:** The site checks whether a username exists.  Then, if username does not exist, it encrypts the password and stores it in the database.  If it matches, it may return an authentication token to the user's client to login.
 ---
 
 - [ ] Imagine that as a user, you are now logging back into that same website. 
 
 * **Question:** How does the website verify that you are indeed the same user?
 
-* **Your Answer:**
+* **Your Answer:** The site checks whether a username exists.  Then, if username exists, it encrypts the password and checks to see if the password matches the username.  If it matches, it returns an authentication token to the user's client.
```

Comment:
In the case of login, it is going to verify that the plain text password matches the encrypted version.